web/server/entrypoint_arm64.go

```diff
@@ -0,0 +1,27 @@
+//go:build cgo
+
+package server
+
+import (
+	"go.viam.com/rdk/gostream"
+	"go.viam.com/rdk/gostream/codec/h264"
+	"go.viam.com/rdk/gostream/codec/opus"
+	"go.viam.com/rdk/gostream/codec/x264"
+	"go.viam.com/rdk/utils"
+	"strings"
+)
+
+var streamConfig gostream.StreamConfig
+
+func init() {
+	if osInfo, err := utils.DetectOSInformation(); err == nil && strings.Contains(osInfo.Device, "Raspberry Pi") {
```

Comment:
Can this not be added to `entrypoint_unix.go`?

Reply:
This line _will_ build on any Unix architecture. The line below it will _not_. Since the `h264` package depends on static libraries built on the arm64, attempting to build it on other platforms will result in arm64 symbols being undefined and the build will fail.